src/drf_yasg/utils.py

```diff
@@ -182,6 +183,8 @@ def SwaggerType(**instance_kwargs):
         if swagger_object_type != openapi.Items and 'default' not in instance_kwargs:
             default = getattr(field, 'default', serializers.empty)
             if default is not serializers.empty:
+                if isinstance(default, CurrentUserDefault):
```

Comment:
reviewing myself again, I'm not sure about the fix. Maybe I should do like DRF is doing...
https://github.com/encode/django-rest-framework/blob/master/rest_framework/fields.py#L253
yeah, seems better.